bindings/python/pymongoarrow/lib.pyx

```diff
@@ -115,6 +119,10 @@ def process_bson_stream(bson_stream, context):
                         if value_t == BSON_TYPE_UTF8:
                             bson_str = bson_iter_utf8(&doc_iter, &str_len)
                             builder.append(<bytes>(bson_str)[:str_len])
+                        elif value_t == BSON_TYPE_DECIMAL128:
+                            bson_iter_decimal128(&doc_iter, &dec128)
+                            bson_decimal128_to_string(&dec128, decimal128_str)
+                            builder.append(<bytes>(decimal128_str))
```

Comment:
Does builder.append take a copy of the decimal128_str?

Reply:
I don't follow, you mean copy vs by reference?  It is coerced to a Python object either way, I was just making it explicit.

Reply:
SGTM